packages/expo-battery/CHANGELOG.md

```diff
@@ -16,7 +18,7 @@
 
 ### 🐛 Bug fixes
 
-- Remove peerDependencies and unimodulePeerDependencies from Expo modules. ([#11980](https://github.com/expo/expo/pull/11980) by [@brentvatne](https://github.com/brentvatne))
+- Remove `peerDependencies` and `unimodulePeerDependencies` from Expo modules. ([#11980](https://github.com/expo/expo/pull/11980) by [@brentvatne](https://github.com/brentvatne))
```

Comment:
Funny, why it is changed 🤔 @Simek, do you manually edited it? 🤔

Reply:
Yup, is it incorrect change?

Reply:
@bbarthec I have reverted the formatting change.

Reply:
That change was ok 😉 I was just curious 🧐 
In the future you can adjust the visual part of CHANGELOGs for the older entires if you like 💪